tests/test_interface.rb

```diff
@@ -586,8 +582,7 @@ def negotiate_auto_helper(interface, default, cmd_ref)
                  "Error: #{inf_name} negotiate auto value not #{default}")
 
     pattern = cmd_ref.test_config_get_regex[default ? 1 : 0]
-    assert_interface_match_line(interface.name, pattern,
-                                "#{inf_name} no negotiate auto cfg mismatch")
+    assert_show_match(pattern: pattern)
```

Comment:
This is where the magic comes in with `@default_show_command` - if you look below, both `test_negotiate_auto_portchannel` and `test_negotiate_auto_ethernet` are setting `@default_show_command`, so if we don't pass a `command` parameter to this method, that default show command is what gets used.

Is this being too clever for my own good?